src/apps/content/templates/includes/header.html

```diff
@@ -11,7 +11,8 @@
             </form>
             {% if user.is_authenticated %}
              <div class="col-md-3 text-end">
-                 <a href="{% url 'user_profile' %}" class="btn btn-outline-primary">Profile</a>
+                 <a href="{% url 'user_detail' %}" class="btn btn-outline-primary">User</a>
```

Comment:
Maybe it would be better to leave Profile name or even better make user.name clickable for this button?

Reply:
done